### ISSUE TITLE
Fields in JSON mappings can no longer be #f

### ISSUE THREAD

**Issue by civodul** [aconchillo/guile-json#69]
Hi!

Starting from version 4.4.1 or thereabouts (the bug was definitely not in 4.3.2), it is no longer possible for a JSON mapping to set a field to `#f` as this is interpreted as "unspecified field", as in this example:

```scheme
(define-json-mapping <foo> make-foo foo?
  json->foo
  (x foo-x "x" (lambda (x)
                 (not (string=? x "fals")))))

(json->foo "{ \"x\": \"fals\" }")
=> #<<foo> x: #<unspecified>>
```

This looks like a bug to me as it should be possible to use any value for fields; WDYT?

(This was originally reported at https://issues.guix.gnu.org/45615.)

**Comment by aconchillo** [comment]
Yes, this is a bug. I think it was introduced when I added support to not serialize `*unspecified*` fields.

**Comment by aconchillo** [comment]
@civodul I believe this should fix it https://github.com/aconchillo/guile-json/pull/70

**Comment by aconchillo** [comment]
This should now be fixed in 4.5.1. Thanks for reporting!

**Comment by civodul** [comment]
Hi @aconchillo!  Thanks for the quick fix and release!

I had to adjust the Guix code to unspecified value handling, but after that switching to 4.5.1 solves the problem.
